src/sentry/replays/usecases/ingest/dom_index.py

```diff
@@ -0,0 +1,145 @@
+import time
+import uuid
+import zlib
+from typing import Iterator, List, Literal, Optional, TypedDict
+
+from django.conf import settings
+
+from sentry.utils import json, kafka_config
+from sentry.utils.pubsub import KafkaPublisher
+
+replay_publisher: Optional[KafkaPublisher] = None
+
+ReplayActionsEventPayloadClick = TypedDict(
```

Comment:
do you have to use the `TypedDict` constructor here?

Reply:
`class` is a reserved keyword.